blueman/config/AutoConnectConfig.py

```diff
@@ -0,0 +1,29 @@
+import gi
+gi.require_version("Gtk", "3.0")
+from gi.repository import Gtk
+
+from blueman.bluez.Device import Device
+from blueman.main.Config import Config
+
+
+class AutoConnectConfig(Config):
+    def __init__(self) -> None:
+        super().__init__("org.blueman.plugins.autoconnect")
+
+    def bind_to_menuitem(self, item: Gtk.CheckMenuItem, device: Device, uuid: str) -> None:
```

Comment:
This is confusing as it is not related to menuitems, more to the services key in gsettings. I would suggest to name it ``bind_service`` instead.